operator/pkg/central/extensions/reconcile_tls_test.go

```diff
@@ -237,3 +374,379 @@ func TestRenewInitBundle(t *testing.T) {
 		})
 	}
 }
+
+func Test_createCentralTLSExtensionRun_validateAndConsumeCentralTLSData(t *testing.T) {
+
+	type testCase struct {
+		fileMap types.SecretDataMap
+		assert  func(t *testing.T, err error)
+	}
+
+	randomCA := func() (mtls.CA, error) {
+		serial, err := mtls.RandomSerial()
+		if err != nil {
+			return nil, errors.New("could not generate serial number")
+		}
+		req := csr.CertificateRequest{
+			CN:           "Planet Express",
```

Comment:
Maybe choose a completely unrealistic name to avoid potential trademark infringement claims? :sweat_smile: